telemetry/pipeliner.go

```diff
@@ -45,12 +45,12 @@ type PipelinedVertex struct {
 	Pipelines []pipeline.Path
 }
 
-func (t *Pipeliner) WriteStatus(ev *progrock.StatusUpdate) error {
```

Comment:
Another side fix; there was no need for this type to be a `progrock.Writer`, and removing the `error` return value makes the caller's lives easier.